day_7/fizzbuzz.rb

```diff
@@ -0,0 +1,14 @@
+puts "Please enter a number to count up to:"
```

Comment:
Great job with fizzbuzz! I would recommend attempting all of the Day-7 challenges to build those ruby muscle!